orgs-terms.md

```diff
@@ -0,0 +1,61 @@
+# npm Orgs Terms
+
+These npm Orgs Terms of Use (these _npm Orgs Terms_) supplement
+the terms for npm Open Source offered by npm, Inc. (_npm_) at
+<https://www.npmjs.com/policies/open-source-terms> (_npm Open Source
+Terms_). They govern access to and use of _npm Orgs_, the organization
+private package storage, delivery, and access control features of
+<https://www.npmjs.com> (the _Website_) and the npm public registry at
+<https://registry.npmjs.com> (the _Public Registry_).
+
+These npm Orgs Terms were last updated on
+November 18, 2015. You can review prior versions at
+<https://github.com/npm/policies/commits/master/orgs-terms.md>.
+
+You may only access or use npm Orgs by agreeing to the npm Open
+Source Terms as supplemented by these npm Orgs Terms. If npm adds any
+additional functionality to npm Orgs, you must agree to these npm Orgs
+Terms to use those new features, too. You add these npm Orgs Terms to
+your agreement with npm by enabling npm Orgs for your account on the
+Website (your _Account_). These npm Orgs Terms then become a part of the
+contract between you and npm, until you or npm disable npm Orgs for your
+Account.
+
+## npm Orgs
+
+npm will provide the features and services described
+in the public documentation for npm Orgs at
+<https://www.npmjs.com/npm/private-packages>, and npm grants you
+permission to use those features and services for one organization (your
+_Org_). Permission to use npm Orgs isn't exclusive to you, and you
+can't transfer it to others. You may give other Website account holders
+permission to use npm Orgs by adding their accounts as members of your
+Org, provided those account holders agree to and abide by the npm Open
+Source Terms.
+
+Your permission to use npm Orgs, your Org members' permission to use npm
+Orgs, and npm's commitment to provide npm Orgs are all subject to these
+npm Orgs Terms, the npm Open Source Terms, and the following additional
+conditions:
+
+1.  When enabling npm Orgs for your Account, you must provide
+    all the payment card details requested by the Website (your _Payment
+    Details_). Those details must be for a valid payment card that you
+    have the right to use (your _Payment Card_). You must keep your
+    Payment Details up-to-date via the Website.
+
+2.  You must pay $7.00 via your Payment Card when you enable npm Orgs or
```

Comment:
The terms should make clear how much we will charge users, for what, and when. Marketing materials should not mislead anyone, but can be more general, provided the nitty-gritty is otherwise available.

If how the product is "counted" for billing terms is itself complex, we have to communicate that complexity. The most natural place to do so is in the terms.

Say I pay for PM for myself, privately, and I want 3 Orgs (A, B, and C) with 1, 2, and 3 additional, otherwise unpaid, users, respectively. In that case "$7/user/month, min. 2[ /org]" alone could be read a few ways. Is my bill?:

| Reading | PM    | A    | B    | C    | Total |
|---------|-------|------|------|------|-------|
| 1       | $7    | $14  | $21  | $28  | $70   |
| 2       | $7    | $7   | $14  | $21  | $49   |
| 3       | $7    | $14  | $14  | $21  | $56   |
| ...     | ?     | ?    | ?    | ?    | ?     |

I believe Reading 3 above is correct. Please let me know, and I'll revise the language.